### ISSUE TITLE
Clean pipeline using Neuraxle

### ISSUE THREAD

**Issue by guillaume-chevalier** [guillaume-chevalier/seq2seq-signal-prediction#15]
Should do something that looks like this to clean the project by using [Neuraxle](https://github.com/Neuraxio/Neuraxle):
```
deep_learning_seq_classif_pipeline = EpochRepeater(Pipeline([
    Windower(),  # TODO: not sure where this must be plugged into the pipeline.
    AutoRegress(),  # TODO: AutoRegress might be inside NormalizeThenUnNormalizeWrapper, not sure.
    NormalizeThenUnNormalizeWrapper(
        TrainOnlyWrapper(DataShuffler(seed=42)),
        MiniBatchSequentialPipeline([
            ForEachDataInput(Pipeline([
                ToNumpy(np_dtype=np.float32),
                DefaultValuesFiller(0.0),
            ])),
            ForecastModel(n_stacked=2, n_residual=3),
        ], batch_size=32),
    )
]), epochs=200, fit_only=True)
```

Where `ForecastModel` contains the actual TensorFlow code.

**Comment by guillaume-chevalier** [comment]
Similar issue: https://github.com/guillaume-chevalier/LSTM-Human-Activity-Recognition/issues/33

**Comment by guillaume-chevalier** [comment]
Solved with #17
